.github/workflows/test.yml

```diff
@@ -30,4 +30,4 @@ jobs:
       - name: doctest
         shell: bash -l {0}
         run: |
-          pytest --doctest-modules echemdb
+          pytest -n auto --doctest-plus --remote-data --doctest-modules echemdb
```

Comment:
@DunklesArchipel @linuxrider you can now use
```
pytest -n auto --doctest-plus --doctest-modules echemdb
```
to run tests more quickly, i.e., skip the tests that download the echemdb database.